src/internal/connector/exchange/event_calendar_cache.go

```diff
@@ -94,7 +94,7 @@ func (ecc *eventCalendarCache) AddToCache(ctx context.Context, f graph.Container
 	}
 
 	temp := graph.NewCacheFolder(f,
-		path.Builder{}.Append(calendarOthersFolder, *f.GetDisplayName()), // storage path
+		path.Builder{}.Append(calendarOthersFolder, *f.GetId()),          // storage path
```

Comment:
this feels so anti-climatic after all the other stuff 😛 

Anyway, there's a few places this needs updated actually. See comments from the previous PR:
* https://github.com/alcionai/corso/pull/2430#discussion_r1099501415
* https://github.com/alcionai/corso/pull/2430#discussion_r1099502588

Reply:
It's the best way to have it happen!